src/components/RightPane/SectionTable/SectionTable.js

```diff
@@ -82,21 +82,16 @@ const SectionTable = (props) => {
                     deptCode={courseDetails.deptCode}
                     courseTitle={courseDetails.courseTitle}
                     courseNumber={courseDetails.courseNumber}
+                    prerequisiteAnchorInfo={{
+                        link: courseDetails.prerequisiteLink,
+                        analyticsAction: analyticsEnum.classSearch.actions.CLICK_PREREQUISITES,
```

Comment:
is this value ever not `analyticsEnum.classSearch.actions.CLICK_PREREQUISITES`? If not, I don't think it needs to be a field in this object, it can just be hard-coded into CouseInfoBar.